src/System.Diagnostics.FileVersionInfo/tests/System.Diagnostics.FileVersionInfo.Tests/Interop/Interop.Windows.cs

```diff
@@ -5,11 +5,14 @@
 using System.Runtime.InteropServices;
 using System.Text;
 
-internal partial class Interop
+namespace System.Diagnostics.Tests
 {
-    internal partial class mincore
+    internal partial class Interop
```

Comment:
Actually, I missed this... this should not be put into a namespace.  Please revert this file.